Tasks/ServiceFabricDeploy/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -25,6 +25,8 @@
   "loc.input.help.skipUpgradeSameTypeAndVersion": "Indicates whether an upgrade will be skipped if the same application type and version already exists in the cluster, otherwise the upgrade fails during validation. If enabled, re-deployments are idempotent.",
   "loc.input.label.skipPackageValidation": "Skip package validation",
   "loc.input.help.skipPackageValidation": "Indicates whether the package should be validated or not before deployment.",
+  "loc.input.label.unregisterUnusedVersions": "Unregister Unused Versions",
+  "loc.input.help.unregisterUnusedVersions": "This will remove all unused type versions after deployment.",
```

Comment:
I would rephrase to "Indicates whether all unused versions of the application type will be removed after an upgrade."